### PR TITLE
Add openwebicons w/ npm auto-update

### PR DESCRIPTION
**Pull request by x09326**
PR for #9221 
@kennynaoh @pvnr0082t 
Please help me review it.
Manually add other versions of openwebicons from git repo because npm auto-update can't get some versions.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/pfefferle/openwebicons
 * Official website (optional, not the repository):https://pfefferle.github.io/openwebicons/
 * NPM package url (optional):https://www.npmjs.com/package/openwebicons
 * GitHub / Bitbucket popularity (required):
   - Count of stars:782
   - Count of watchers:105
   - Count of forks:160
 * NPM download stats (optional):
   - Downloads in the last day:6
   - Downloads in the last week:14
   - Downloads in the last month:86

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/openwebicons/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "css",
+    "sass",
+    "scss"
+  ],
+  "npmName": "openwebicons",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "font/*",
```
Comment (pvnr0082t) [inline ajax/libs/openwebicons/package.json]: I think it will be more flexible to use `"font/**/*"`

**Comment by kennynaoh** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Hey @x09326 @pvnr0082t what about the other css files? This is not a trivial solution filemap, for this kind of cases, please DO NOT easily say it's good to you, there should be an explanation, please don't make reviewer guessing what do you thinking.

**Comment by x09326** [comment]
@pvnr0082t 
I have checked the files with author and rebased it. 
[ref](https://github.com/pfefferle/openwebicons/issues/15)
Please help me review it again, thanks.

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**